prisma/schema.prisma

```diff
@@ -14,7 +14,7 @@ datasource db {
 model User {
   discordId     String        @id
   username      String
-  discriminator String
+  discriminator String?
```

Comment:
Prisma is an ORM layer using auto generated code based on the schema. Please create a database migration, you can do this using `npx prisma migrate dev`, provided you have a local database running, and prisma can connect to it.

Reply:
Oh i see. I'll work on that when i get back home

Reply:
It took me a while to get the docker db running. But i'm fairly sure i got it now.
Please manually verify this though. I have 0 experience with prisma, and very little with docker. And i don't want to accidentally break prod and have it `rm -rf db`

Reply:
No need to test it, this looks good.